meetings/2025-05-15/agenda.md

```diff
@@ -0,0 +1,38 @@
+# Agenda for Safety Critical Rust Consortium Meeting, 15 May 2025, Utrecht, Netherlands
+
+The second meeting of the Safety Critical Rust Consortium meeting will be held on 15 May 2025 from 9 AM - 3 PM GMT/UTC in Utrecht, Netherlands. 
+
+Below is the proposed agenda. Additional suggestions welcome; please suggest a change or comment on the pull request associated with this agenda item.
+
+Lunch will be provided to the meeting room.
+
+If we have extra time, we will review open issues and pull requests to see if we can start closing any of those out.
+
+- **9 AM**: Welcome and Meeting Logistics (@JoelMarcey)
+  - Welcome New Members
+- **9:15 AM**: Subsidizing costs of in-person meetings (@JoelMarcey)
+  - Try to have consortium members host in cities where we have meetings?
+- **9:30 AM**: Website for consortium information and status (@JoelMarcey)
+  - Domain
+  - Front matter content
+  - Subcommittee content
```

Comment:
I would like to add a discussion item about how we would like to proceed with publishing what the consortium does?
- do we stay with the repository
- do we migrate it to the arewesaftycriticalyet.org website?